routing/pathfind.go

```diff
@@ -461,10 +476,16 @@ func findPath(tx *bolt.Tx, graph *channeldb.ChannelGraph,
 	additionalEdges map[Vertex][]*channeldb.ChannelEdgePolicy,
 	sourceNode *channeldb.LightningNode, target *btcec.PublicKey,
 	ignoredNodes map[Vertex]struct{}, ignoredEdges map[uint64]struct{},
-	amt lnwire.MilliSatoshi, feeLimit lnwire.MilliSatoshi,
+	amt lnwire.MilliSatoshi, pathRestrictions PathRestrictions,
 	bandwidthHints map[uint64]lnwire.MilliSatoshi) ([]*channeldb.ChannelEdgePolicy, error) {
 
-	var err error
+	var (
+		err error
+	)
+
+	feeLimit := pathRestrictions.FeeLimit
```

Comment:
move these inside the `var ( .. )` as well?

Reply:
Any reason? They're initialised there.